src/tests/unittests/asn1.c

```diff
@@ -47,6 +47,61 @@ static void torture_large_oid(void **state)
 	assert_int_equal(rv, SC_ERROR_NOT_SUPPORTED);
 }
 
+static void torture_oid(void **state)
+{
+	/* (without the tag and length {0x06, 0x06}) */
+	/* Small OIDs */
+	u8 small[] = {0x01, 0x02, 0x03, 0x04, 0x05, 0x06};
+	/* Limit what we can fit into the first byte */
+	u8 limit[] = {0x7F};
+	/* The second octet already oveflows to the second byte */
+	u8 two_byte[] = {0x81, 0x00};
+	/* Missing second byte, even though indicated with the first bit */
+	u8 missing[] = {0x81};
+	/* Missing second byte, even though indicated with the first bit */
+	u8 ecpubkey[] = {0x2A, 0x86, 0x48, 0xCE, 0x3D, 0x02, 0x01};
```

Comment:
the comment doesn't match the value - this is a well-formed encoding of an (1, 2, 840, 10045, 2, 1) OID

Reply:
I think this got fixed on the way, even though I did not comment on this particular issue yet.